application/comit_node/src/swap_protocols/rfc003/ethereum/contract_templates/hex/erc20_contract.asm.hex

```diff
@@ -0,0 +1 @@
+635000000560203614602060006000376020602160206000600060026048f17f100000000000000000000000000000000000000000000000000000000000000160215114161661005b57420363200000021061009e5760006000f35b7fb8cac300e37f03ad332e581dea21b2f0b84eaaadc184a295fef71e81f44a741360006000a17330000000000000000000000000000000000000036020526100e1565b7f5d26862916391bf49478b2f5103b0720a842b45ef145a268f2cd1fb2aed5517860006000a17340000000000000000000000000000000000000046020526100e1565b63a9059cbb6000527f5000000000000000000000000000000000000000000000000000000000000005604052602060606044601c6000736000000000000000000000000000000000000006620186a05a03f150602051ff
```

Comment:
:question: Is the lack of newline at the end of these hex files intentional?

Reply:
It is generated without a newline by `solc` so I cannot really help it.
Also, a newline is likely to mess-up the transaction format so I think it's ok to keep it like that.